index.html

```diff
@@ -2001,44 +2001,73 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 It is expected that securing mechanism specifications other than the ones
-referred to above will be created as necessary. A securing mechanism
-specification:
+referred to above will be created as necessary.
         </p>
 
-        <ul>
-          <li>
-MUST document normative algorithms that provide content integrity protection
-for <a>conforming documents</a>. The algorithms MAY be general in nature and
-MAY be used to secure data other than <a>conforming documents</a>.
-          </li>
-          <li>
-MUST provide a verification mechanism that returns only the information in the
-<a>conforming document</a> that has been secured. It MAY provide an
-additional mechanism to receive other information that might be helpful
-during validation or for debugging purposes. The concrete interface that is
-expected to be provided is detailed in the
-<a href="#verify-securing-mechanism">securing mechanism verification
-algorithm</a> section.
-          </li>
-          <li>
-MUST document post-processing considerations for data returned from verification
-processes. This includes, but is not limited to, what information has been
-secured, for what period of time after verification, when or what additional
-information might be required to assist in decision making, and when
-care is necessary even when the returned data was properly secured.
-          </li>
-          <li>
-SHOULD provide integrity protection for any information referenced by a URL that
-is critical to validation. Mechanisms that can achieve this protection are
-discussed in Section <a href="#integrity-of-related-resources"></a> and Section
+        <p>
+Securing mechanism specifications MUST document normative algorithms that
+provide content integrity protection for <a>conforming documents</a>. The
+algorithms MAY be general in nature and MAY be used to secure data other than
+<a>conforming documents</a>.
+        </p>
+
+        <p>
+Securing mechanism specifications MUST provide a verification mechanism that
+returns only the information in the <a>conforming document</a> that has been
+secured. It MAY provide an additional mechanism to receive other information
```

Comment:
How does this sentence apply when Data Integrity is used?  Does it mean that the "proof" member must not be returned in the secured information?  That would be my interpretation but this could be clearer.

Reply:
All current Data Integrity specifications secure all the information in the `proof` value. There is no such thing as "unprotected" in Data Integrity. 

That said, we defer what to do to the securing mechanism specifications. It is their remit wrt. what is returned. For example, vc-jose-cose might choose to strip out `proof` values that were not verified during the securing process (but by doing that, it'll make it impossible for upstream processors to verify those proofs). So, there will need to be work done in vc-jose-cose to determine what to do there.

For Data Integrity, I expect all the current cryptosuites we have to return `proof` in the returned data because all information in the `proof` values ARE protected and verified as a part of the securing mechanism verification process.

Reply:
In DI, all verified proofs are secured -- so DI securing mechanisms can return all proofs there were verified using the verifier's accepted cryptosuites from the base verification algorithm.

Reply:
The plain meaning of the text above implies that `proof` is used to secure the secured data and is not in the secured data.  Please be clear that `proof`, if present, must be removed by this algorithm.

Reply:
@selfissued fixed in 8c16fec9990f0f5b4b0d16d1fd8d587fa59a5461.